app/models/operatingsystems/solaris.rb

```diff
@@ -89,6 +89,7 @@ def jumpstart_params(host, vendor)
     server_ip   = host.domain.resolver.getaddress(server_name).to_s
     jpath       = jumpstart_path host.medium, host.domain
     ipath       = interpolate_medium_vars(host.medium.media_dir, host.architecture.name, self)
+    network      = host.subnet.network
```

Comment:
This might have to look at `host.provisioning_interface.subnet.network`, as the provisioning interface may be different to the primary.